sdk/examples/how_tos/advanced_transactions/send_micro_transaction.rs

```diff
@@ -26,14 +26,15 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
+    // Get the wallet we generated with `create_wallet`.
```

Comment:
No create_wallet and get anymore :trollface:

Reply:
but there's still a `create_wallet` example or what do you mean?

Reply:
Ah it references the example and not a fn, wasn't clear to me from reading this